department-of-veterans-affairs/abd-vro
# Feature - add endpoint to classify multi contention VA.gov claims

## What was the problem?


Associated tickets or Slack threads:
- Part 2 of https://github.com/department-of-veterans-affairs/vagov-claim-classification/issues/152

## How does this fix it?[^1]

- adds `/va-gov-claim-classifier` to take accept and classify multi-contention claims
- added deprecated=True to /classifier which greys it out in the documentation


## How to test this PR
- new tests in `tests/` dir
- run `pytest`
- additionally, running the code locally, API can be tested manually via POST request

[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
